### PR TITLE
Add ability to set witness proxy voter, #464

### PR DESCRIPTION
**Pull request by svk31**

### CODE REVIEW

**Comment by sneak** [comment]
@svk31 please add the translation keys to the other localization files too with their english texts and tag them with a FIXME comment.

**Comment by sneak** [comment]
@svk31 i love additional functionality but this one is only useful to a very small community of whales, witnesses, bitshares holdovers, et c. and doesn't move the needle much on user acquisition/retention which is our primary focus.  See Trello for priorities regarding those.

File: app/locales/en.js [hunk 1]
```diff
@@ -388,6 +388,12 @@ const en = 	{
 	witness_thread: 'witness thread',
 	you_have_votes_remaining: 'You have {votesCount} votes remaining',
 	you_can_vote_for_maximum_of_witnesses: 'You can vote for a maximum of 30 witnesses',
+	set_witness_proxy: "You can also choose a proxy that will vote for witnesses for you. This will reset your current witness selection.",
```
Comment (sneak) [inline app/locales/en.js]: please add these (as english, tagged with a `FIXME` comment) to the other language files too.

**Comment by svk31** [comment]
I've added the strings as requested.

**Pull request merged**